source/game/kart/KartMove.cc

```diff
@@ -390,6 +390,7 @@ void KartMove::calcDirs() {
     EGG::Vector3f right = dynamics()->mainRot().rotateVector(EGG::Vector3f::ex);
     EGG::Vector3f local_88 = right.cross(m_smoothedUp);
     local_88.normalise();
+    m_b东西 = true;
```

Comment:
Can you at least try naming this? It makes it easier to iteratively improve variable naming if we have _something_ to work off of

Reply:
i can try